lib/model/model_test.go

```diff
@@ -1779,7 +1779,7 @@ func TestIssue3164(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	fl := rwFolder{
+	fl := sendreceiveFolder{
```

Comment:
Should I rename all sendreceiveFolder to sendReceiveFolder, even in rwfolder.go ?